UnityProject/Assets/Scripts/Health/HealthBehaviour.cs

```diff
@@ -56,37 +56,40 @@ public void ServerOnlySetHealth(int newValue)
 		}
 	}
 
-	///fixme/todo: to be replaced by net messages, crappy and unsecure placeholder
-	[ClientRpc]
-	public void RpcApplyDamage(string damagedBy, int damage, DamageType damageType, BodyPartType bodyPartAim)
-	{
-		if (isServer || !isNPC || IsDead)
-		{
-			return;
-		}
-		ApplyDamage(damagedBy, damage, damageType, bodyPartAim);
-	}
-
-	public void ApplyDamage(string damagedBy, int damage, DamageType damageType, BodyPartType bodyPartAim = BodyPartType.CHEST)
-	{
-		if (damage <= 0 || IsDead)
-		{
-			return;
-		}
-		int calculatedDamage = ReceiveAndCalculateDamage(damagedBy, damage, damageType, bodyPartAim);
+    ///fixme/todo: to be replaced by net messages, crappy and unsecure placeholder

```

Comment:
Well many things are still crappy insecure placeholders for 0.2.2, if it works it works.

Reply:
actually this note was left by me, a few months ago :)